### PR TITLE
Change logging to use .log file, add http POST logging

### PR DESCRIPTION
**Pull request by Termuellinator**
I've changed the logger settings to use .log instead of .txt and added HTTP POST logging to the custom logging middleware.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 # log files
 logs/*.txt
+log/*.log
```
Comment (Swetlana-K) [inline .gitignore]: typo, should be logs

**Review by Swetlana-K — CHANGES_REQUESTED** [review]
pleas fix the typo

File: .gitignore (hunk 1 above)
Comment (Termuellinator) [reply .gitignore]: Oh, sorry - fixed it!

**Review by Swetlana-K — APPROVED** [review]
okay

File: .gitignore (hunk 1 above)
Comment (Swetlana-K) [reply .gitignore]: LGTM

**Comment by Swetlana-K** [comment]
Danke für die Hilfe

**Pull request merged**